Dockerfile

```
@@ -0,0 +1,57 @@
+#-------------------------------#
+# Development/build environment #
+#-------------------------------#
+
+FROM  debian:11.2 as taste-dev-env
+
+# Meta
+LABEL VENDOR="N7 Space"
+LABEL ARTEFACT="TASTE Development Environment"
+LABEL DESCRIPTION="Build environment for additional TASTE tools"
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DOTNET_CLI_TELEMETRY_OPTOUT=true
+ENV PATH="/asn1scc/asn1scc/bin/Debug/net5.0/:${PATH}"
+
+# Setup apt dependencies
+RUN apt-get update && apt-get install -y \
```

```suggestion
RUN apt-get update && apt-get install -qy --no-install-recommends \
```
some apt magics found in "docker best practices"

---

oh, and -q to udpate also

---

ack